tasks/request_from_archive/request_from_archive.py

```diff
@@ -583,32 +585,74 @@ def restore_object(
         recovery_type: Valid values are
             'Standard'|'Bulk'|'Expedited'.
     Raises:
-        ClientError: Raises ClientErrors from restore_object, or if the file is already restored.
+        None
     """
     request = {"Days": days, "GlacierJobParameters": {"Tier": recovery_type}}
     # Submit the request
     restore_result = s3_cli.restore_object(
         Bucket=db_archive_bucket_key, Key=key, RestoreRequest=request
-    )  # Allow ClientErrors to bubble up.
+    )
     if restore_result["ResponseMetadata"]["HTTPStatusCode"] == 200:
-        # Will have a non-error path after https://bugs.earthdata.nasa.gov/browse/ORCA-336
-        raise ClientError(
-            {
-                "Error": {
-                    "Code": "HTTPStatus: 200",
-                    "Message": f"File '{key}' in bucket "
-                    f"'{db_archive_bucket_key}' has already been recovered.",
-                }
-            },
-            "restore_object",
-        )
+        LOGGER.info(f"File '{key}' in bucket '{db_archive_bucket_key}' has already been recovered"
+                    "Sending to archive recovery SQS")
+        archive_recovery_queue_url = str(os.environ[OS_ENVIRON_ARCHIVE_RECOVERY_QUEUE_URL_KEY])
+
+        post_to_archive_queue(archive_recovery_queue_url, key, db_archive_bucket_key)
 
     LOGGER.info(
         f"Restore {key} from {db_archive_bucket_key} "
         f"attempt {attempt} successful. Job ID: {job_id}"
     )
 
 
+def post_to_archive_queue(archive_recovery_queue_url: str, key, bucket_name: str) -> None:
```

Comment:
Use the pre-build `post_to_standard_queue` function in shared_recovery to avoid rewriting error-handling. See https://github.com/nasa/cumulus-orca/blob/960435497c717f219a335372791b25dcb9fb87eb/tasks/post_copy_request_to_queue/post_copy_request_to_queue.py#L116 for example.

Reply:
aah I forgot about that function. Looks like I can just use that to post to queue.